superset-frontend/src/features/alerts/components/AlertReportCronScheduler.test.tsx

```diff
@@ -32,7 +32,7 @@ const createProps = (props: Partial<AlertReportCronSchedulerProps> = {}) => ({
   ...props,
 });
 
-test('should render', () => {
+test.skip('should render', () => {
```

Comment:
Can we unskip/fix those tests?

Reply:
These tests are artifacts from the previous modal design left for reference during review. They should be removed before submitting the PR.